osclib/accept_command.py

```diff
@@ -254,7 +255,7 @@ def create_new_links(self, project, pkgname, oldspeclist):
 
         if len(filelist) > 1:
             # There is more than one .spec file in the package; link package containers as needed
-            origmeta = source_file_load(self.api.apiurl, project, pkgname, '_meta')
```

Comment:
source_file_load is our own function - and I would say it always should return a `str`

Reply:
makes sense to me.